test/hotspot/jtreg/runtime/ErrorHandling/MachCodeFramesInErrorFile.java

```diff
@@ -71,7 +71,8 @@ public static void main(String[] args) throws Exception {
                 crashInJava1(10);
             } else {
                 assert args[0].equals("crashInVM");
-                crashInNative1(10);
+                // AIX does not prohibit low address reads
+                crashInNative1(Platform.isPPC() ? -1 : 10);
```

Comment:
Is it AIX or PPC or AIX/PPC that needs the change? The code should match the comment.

Reply:
I remember changing this now. It's actually PPC. The Power ISA specifies that the first 256 bytes are 'reserved for software use'.

I'll change the comment.

Reply:
Really? I was always assuming the linux kernel handles this differently.

Reply:
I think the mentioned Power ISA chapter addresses real addresses, not the user mode virtual address space. We always get SIGSEGV when accessing the first page on linux.

Reply:
Thanks for clarifying!